Rakefile

```diff
@@ -19,10 +19,17 @@ task :"test:envs", [:grep] do |t, args|
   end
 end
 
-task :up do
-  sh "docker-compose up -d | cat"
+namespace :docker do
+  task :up do
+    sh "docker-compose up -d | cat"
+  end
+  
+  task :down do
+    sh "docker-compose kill"
```

Comment:
we can use ``docker-compose down`` instead of kill. It will delete the containers after killing them (not the image)